### ISSUE TITLE
ARIN - Could not validate protocol response after update to 0.6.2

### ISSUE THREAD

**Issue by christheradioguy** [NLnetLabs/krill#240]
Hello,

I am currently using Krill with delegated RPKI from ARIN and recently updated krill to version 0.6.2.  Since upgrading I have noticed that all my ROAs have been withdrawn and I am seeing the following error in my log file:
Could not validate protocol response: MIAGCSqGSIb3DQEHA.......
Failed to refresh CA certificates for CA Name, error: validation error

I upgraded by backing up my data folder and config, doing a git pull and then rebuilding with cargo.  Anything I can do to resolve this issue?

**Comment by timbru** [comment]
That is strange.. we tested the upgrade under the RIPE NCC parent ourselves and did not have any such issues.

The message 'Could not validate protocol response' indicates that ARIN is using a different key to sign its messages from the one your Krill expects.

Are you sure that you were using the latest data directory and config, and that you started up Krill with that configuration file as well?

It looks like you went back to a prior state. ROAs disappearing also point to that - because - unless you have a successful exchange with your parent and they tell you that you lost resources on a ROA, it will just be kept when this exchange fails.

You can check your Krill history using this command, which may help to see what happened:
`krillc history [--ca <myca>]`

Can you please double check and let me know?

**Comment by christheradioguy** [comment]
It was very strange behavior, it's like ARIN removed the delegation or something, I observed that no traffic was coming in to my rsync server, and when I ran RIPE's RPKI validator I didn't see my server in the Trust Anchor.  All my ROAs were still present in the Krill UI though, and it would let me create new ROAs for my certified resources as well.

I ended up blowing away the old configuration and re-delegating RPKI in ARIN and everything is working as expected again.
